Dockerfile

```diff
@@ -124,6 +124,16 @@ RUN gem install ronn --no-ri --no-rdoc \
   && chmod a+x /usr/local/bin/php-cs-fixer \
   && cd /
 
+# Set up tools for Python codegen; these are:
+#   pandoc: an apt package that can convert text between formats
+#     (example: Markdown to Restructured Text)
+#   protoc-docs-plugin: A protoc plugin to add docstrings to the Python
+#     protoc output.cd
+RUN apt-get update \
+  && apt-get install -y pandoc \
```

Comment:
Exactly. The way protoc works is that `--foo_out={place}` translates to running `protoc-gen-foo` in `$PATH`.